docs/website/docs/general-usage/incremental-loading.md

```diff
@@ -878,6 +878,51 @@ Consider the example below for reading incremental loading parameters from "conf
    ```
    `id_after` incrementally stores the latest `cursor_path` value for future pipeline runs.
 
+### Loading NULL values in the incremental cursor field
+
+When loading incrementally with a cursor field, each row is expected to contain a value at the cursor field that is not `None`.
+For example, the following source data will raise an error:
+```py
+data = [
+    {"id": 2, "updated_at": 2},
+    {"id": 1, "updated_at": 1},
+    {"id": 3, "updated_at": None},
+]
+```
+
+If you want to load data that includes `None` values you can transform the records before the incremental processing.
+You can add steps to the pipeline that [filter, transform, or pivot your data](../general-usage/resource.md#filter-transform-and-pivot-data).
+It is important to use the `insert_at` parameter to control the order of the execution and ensure that your custom steps are executed before the incremental processing starts.
```

Comment:
Let's put information about inset_at under TIP or CAUTION. Otherwise it's hard to realise where the insert_at came from
```suggestion
:::caution
It is important to set the `insert_at` parameter of `add_map` function to control the order of the execution and ensure that your custom steps are executed before the incremental processing starts.
In the following example, the step of data yielding is at `index = 0`, the custom transformation at `index = 1`, and the incremental processing at `index = 2`.
:::
```